dbfit-java/core/src/main/java/dbfit/api/DbTable.java

```diff
@@ -30,8 +30,8 @@ public DbTable(DBEnvironment dbEnvironment, String tableName)
 
     public StatementExecution buildPreparedStatement(
```

Comment:
> Should we rename this to buildStatementExecution?

Good question about the naming... It's perhaps used in other places, and it's still kind of prepared statement - not so bad name.

Another wondering was about this extra `Execution` - it makes names a bit longer. Would it be cleaner to drop it from method names or it's not worth.